### PR TITLE
FIX: websiteGet 

### PR DESCRIPTION
**Pull request by nicolas2bert**
Fixing websiteGet feature.

### CODE REVIEW

File: lib/routes/routesUtils.js [hunk 1]
```diff
@@ -428,19 +428,30 @@ const routesUtils = {
...
         const redirectProtocol = protocol || encrypted ? 'https' : 'http';
         const redirectCode = httpRedirectCode || 302;
+        const redirectHostName = hostName || hostHeader;
 
-        let redirectKey = objectKey === undefined ? '' : objectKey;
+        let redirectKey;
+        if (objectKey === undefined) {
+            redirectKey = '';
+        } else if (indexDocument) {
+            redirectKey = objectKey.replace(indexDocument, '');
```
Comment (electrachong) [inline lib/routes/routesUtils.js]: Does replacing the index key with an empty string fix some tests? Which tests were not passing with the previous behavior?

File: lib/routes/routesUtils.js [hunk 2]
```diff
@@ -428,19 +428,30 @@ const routesUtils = {
...
 
         const redirectProtocol = protocol || encrypted ? 'https' : 'http';
         const redirectCode = httpRedirectCode || 302;
+        const redirectHostName = hostName || hostHeader;
 
-        let redirectKey = objectKey === undefined ? '' : objectKey;
+        let redirectKey;
+        if (objectKey === undefined) {
+            redirectKey = '';
+        } else if (indexDocument) {
```
Comment (electrachong) [inline lib/routes/routesUtils.js]: missing indentation and should be `else if` right?

Comment (nicolas2bert) [inline lib/routes/routesUtils.js]: no worries - thanks for the review :)

**Comment by electrachong** [comment]
Opened a new PR https://github.com/scality/S3/pull/483 with some modifications so that we do not pass modified object keys to the redirect functions in the first place (this way we do not need to pass indexDocument through several functions just to remove it again). Closing this pull request.

**Pull request closed without merging**